test/core/client_channel/lb_policy/pick_first_test.cc

```diff
@@ -292,10 +308,12 @@ TEST_F(PickFirstTest, ShufflingDisabled) {
   constexpr static size_t kMaxAttempts = 5;
   for (size_t attempt = 0; attempt < kMaxAttempts; ++attempt) {
     absl::Status status = ApplyUpdate(
-        BuildUpdate(kAddresses, MakePickFirstConfig(true)), lb_policy_.get());
+        BuildUpdate(kAddresses, MakePickFirstConfig(false)), lb_policy_.get());
```

Comment:
This needs to pass `true` to `MakePickFirstConfig()`.  The goal of this test is to show that if the env var is not set, then the config parameter is ignored.

Please rename the test to `ShufflingDisabledViaEnvVar` to make this clear.  Also, please add a TODO indicating that this test should be removed when the env var protection is removed.

Reply:
Done